### DISCUSSION TITLE
ChamberLock very mad, no likey

### DISCUSSION THREAD

**Discussion by dsagfa** [BEEmod/BEE2-items#4370] [Tech Support / Q&A]
I'm trying to place a ChamberLock in my test, but it keeps nagging me about some voxels it wants filled above, below, to the left and right of it, but when I fill it, it's still mad. How do I fix this? Am I doing something horribly wrong?

**Comment by TeamSpen210** [comment]
It also requires an open space on the other side of the wall, for the door to lead to. I should add guide images to show how it should be placed.
